components/org.wso2.carbon.identity.application.authenticator.hypr/src/main/java/org/wso2/carbon/identity/application/authenticator/hypr/HyprAuthenticatorConstants.java

```diff
@@ -0,0 +1,178 @@
+                "Authentication failed when extracting the request ID provided by the HYPR server upon " +
+                        "initiating the send push notification request."),
+        HYPR_BASE_URL_INVALID_FAILURE("HYPR-65008", "Provided HYPR base URL is invalid."),
+        HYPR_APP_ID_INVALID_FAILURE("HYPR-65009", "Provided HYPR app ID is invalid."),
+        HYPR_ENDPOINT_API_TOKEN_INVALID_FAILURE("HYPR-65010",
+                "Provided HYPR endpoint API token is either invalid or expired");
+
+        private final String code;
+        private final String message;
+
+        /**
+         * Create an Error Message.
+         *
+         * @param code    Relevant error code.
+         * @param message Relevant error message.
+         */
+        ErrorMessages(String code, String message) {
+
+            this.code = code;
+            this.message = message;
+        }
+
+        /**
+         * To get the code of specific error.
+         *
+         * @return Error code.
+         */
+        public String getCode() {
+
+            return code;
+        }
+
+        /**
+         * To get the message of specific error.
+         *
+         * @return Error message.
+         */
+        public String getMessage() {
+
+            return message;
+        }
+
+        @Override
+        public String toString() {
+
+            return String.format("%s  - %s", code, message);
+        }
+    }
+
+    /**
+     * Includes the HTTP header parameters.
+     */
+    public static class HTTP {
+
+        // HTTP header parameters
+        public static final String AUTHORIZATION = "Authorization";
+        public static final String BEARER = "Bearer ";
+        public static final String CONTENT_TYPE = "Content-Type";
+        public static final String APPLICATION_JSON = "application/json";
+    }
+
+    /**
+     * Includes the HYPR authentication and registration related constants.
+     */
+    public static class HYPR {
+
+        public static final String AUTHENTICATOR_NAME = "HYPRAuthenticator";
+        public static final String AUTHENTICATOR_FRIENDLY_NAME = "HYPR";
+        public static final String SESSION_DATA_KEY = "sessionDataKey";
+        public static final String USERNAME = "username";
+        public static final String AUTH_STATUS = "authStatus";
+        public static final String AUTH_REQUEST_ID = "authRequestId";
+
+        //Configurable parameters
+        public static final String BASE_URL = "baseUrl";
+        public static final String APP_ID = "appId";
+        public static final String HYPR_API_TOKEN = "apiToken";
+
+        // API Parameters
+        public static final String MACHINE = "machine";
+        public static final String MACHINE_ID = "machineId";
+        public static final String REQUEST_ID = "requestId";
+        public static final String RESPONSE = "response";
+        public static final String NAMED_USER = "namedUser";
```

Comment:
```suggestion
        public static final String USERNAME = "username";
```
WDYT?

Reply:
The "namedUser" is the parameter name used in the HYPR API. Hence, we can't change it.